README.rst

```diff
@@ -1,19 +1,18 @@
 Pipfile: the replacement for `requirements.txt`
 ===============================================
 
-``Pipfile`` and its sister ``Pipfile.lock`` are a replacement for the existing standard `pip`_'s ``requirements.txt`` file.
+``Pipfile`` and its sister ``Pipfile.lock`` are designed as a proposed replacement for an existing standard: `pip`_'s ``requirements.txt`` file.
```

Comment:
Maybe say an existing "format"? requirements.txt is not really a standard.

Reply:
Done!